interview-questions.md

```diff
@@ -7,44 +7,55 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. As a developer, I am creating a Rails application with a model called Cohort that has_many students, but OOPS! I forgot to add the foreign key. How can I fix this mistake? What is the name of the foreign key? Would the foreign key be on the Cohort model or the Student model?
 
-  Your answer:
+  Your answer: This invloves associations in the Cohort Model you would have a has_many relationship. A cohort has many students.
 
-  Researched answer:
+  Researched answer: An association is a connection between two Active Record models. Rails supports 6 types of associations.
 
 
 
 2. Which RESTful routes must always be passed params? Why?
 
-  Your answer:
+  Your answer: GET,PATCH,DELETE will require a params. In the form of :id.  
 
-  Researched answer:
+  Researched answer: ID will then point to the given value when an ID is passed in the route. Example: 'student/2' the student with an ID of 2.
 
 
 
 3. Name three rails generator commands. What is created by each?
 
-  Your answer:
+  Your answer:The 3 generate commands I've used have been Rails g Model, Rails g Controlller,Rails g resource
 
-  Researched answer:
+  Researched answer:Rails g Model: Creates a Model, Rails g Controlller: creates a Controller, Rails g resource: Generates a resource.
 
 
 
 4. Consider the Rails routes below. What is the name of the controller method that would be called by each route? What action would each of the controller methods perform?
-
-action: "GET"    location: /students          
-
+   <!-- this will call the students controller and return all students  -->
+action: "GET"    location: /students
+       <!-- This will fire up the POST method in the controller -->
```

Comment:
Post is an HTTP verb, not the standard name for the controller method.  I would recommend revisiting the [RESTful routes](https://github.com/learn-academy-2022-bravo/Syllabus/blob/main/rails/restful-routes-crud.md) section of the syllabus.